cmd/otelcol/config/collector/gateway_config.yaml

```
@@ -130,6 +135,10 @@ service:
       processors: [memory_limiter, batch, resourcedetection/internal]
       exporters: [signalfx/internal]
     logs:
-      receivers: [otlp, signalfx]
+      receivers: [signalfx]
       processors: [memory_limiter, batch]
       exporters: [signalfx]
+    logs/hec:
+      receivers: [otlp, splunk_hec/logs]
+      processors: [ memory_limiter, batch ]
+      exporters: [ splunk_hec/logs ]
```

Just for consistency

```suggestion
      processors: [memory_limiter, batch]
      exporters: [splunk_hec/logs]
```